news/2018-07-01-osu-idol-2018-auditions-now-open.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "osu!idol 2018 Auditions Now Open"
+date: 2018-07-01 00:00:00 +0000
+---
+
+May I interest you in some singing? Want to compete with others to become the next osu!idol? Then grab a mic and join in, it’s that time of year again to submit your entries and show us what you've got!
+
+![](/wiki/shared/news/2018-07-01-osu-idol-2018-auditions-now-open/banner.jpg)
+
+### Hold on, what is this?
+
+The annual community driven singing contest - osu!idol! It's where contestants as a solo or a duet sing their way through multiple stages of different genres with the aim to reach the very top of the idol podium! The panel of judges we have prepared will evaluate your performances and ensure the best is standing at the final stretch. After that, you, the community, will decide with a voting to determine our next osu!idol! It's go time! 
+
+### What should I do?
+
+All you need to do is make a small recording of around 30 seconds with your best performance without any background music. We'll be taking submissions up to a month so no theres no need to rush! Practice and make most of your only shot, otherwise you'd have to wait for next year!
```

Comment:
"We'll be taking submissions up to a month" -> we'll be taking submissions for a month starting today

also "so no theres no need to rush" -> so there's no need to rush

Reply:
Applied